index.html

```diff
@@ -16,23 +16,23 @@
                 <div class="my-logo">
                   <a class="Logo" href="#">My Logo</a>
                 </div>
-                <ul class="nav-list">
-                  <li class="nav-item"><a href="#">portfolio</a></li>
-                  <li class="nav-item"><a href="#">About</a></li>
-                  <li class="nav-item"><a href="#">Contact</a></li>
-                </ul>
                 <div class="hambuger">
                   <div class="bar"></div>
                   <div class="bar"></div>
                   <div class="bar"></div>
                 </div>
```

Comment:
### OPTIONAL


Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_479](https://user-images.githubusercontent.com/69638013/194914602-2fe90a68-584e-4a9b-bc2e-29ea43f11ca6.png)  | ![Screenshot_480](https://user-images.githubusercontent.com/69638013/194915004-b0ed97af-1969-477b-8b42-96e8f01ffa5c.png)


- [ ] Kindly align the logo and hamburger icon in your header to look like the Figma design 👍🏿

Reply:
Thank you for your feedback! I will adjust the alignment of the logo and hamburger icon in the header to match the Figma design as closely as possible.

Thanks again for your help!